taxcalc/functions.py

```diff
@@ -852,7 +889,9 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517,
 
     if (puf and ((_standard == 0 or (_exact == 1 and e04470 > 0)) and
                  f6251 == 1)):
-        _cmbtp = _cmbtp_itemizer
+        _cmbtp = (-1 * min(_edical, 0.025 * max(0., e00100)) + e62100 +
```

Comment:
@GoFroggyRun, _cmbtp_itemizer is only equivalent to the expression here when the extrapolation is not being used. _cmbtp is an imputation of the the AMT preference items that we don't have directly on the puf. The imputation is calculated as the difference between tax-payer given AMTI and calculated AMTI. To get this right, we think we need to calculate _cmbtp with the 09 data (before the extrapolation of all of the variables that make up _cmbtp) and then blowup _cmbtp as a whole at the personal income (ATXPY) blowup factor. 

fyi, _cmbtp stands for "combined tax preferences". 

cc @martinholmer